kong/db/init.lua

```diff
@@ -570,6 +570,14 @@ do
       end
     end
 
+    if run_teardown then
+      ok, err = self.connector:post_run_teardown_migrations()
+      if not ok then
+        self.connector:close()
+        return nil, prefix_err(self, err)
+      end
+    end
```

Comment:
When both `run_up` and `run_teardown` are given, this will wait for schema consensus twice (and print the verbose logs twice as well). Will update locally.